NLog.Windows.Forms/RichTextBoxTarget.cs

```diff
@@ -209,34 +212,59 @@ public RichTextBoxTarget()
         /// </summary>
         protected override void InitializeTarget()
         {
+
             if (FormName == null)
             {
+                InternalLogger.Info("FormName not set, creating a new form");
                 FormName = "NLogForm" + Guid.NewGuid().ToString("N");
             }
-
-            var openFormByName = Application.OpenForms[FormName];
-            if (openFormByName != null)
+            else
```

Comment:
This else is new.  The above used to just create a new form name then line 221.  So the logic above is different.